<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>352</width>
      <height>288</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 8, 8);</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>brightness :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="brightness_slider">
         <property name="minimum">
          <number>-16</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="minimum">
          <number>-16</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>hue               :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hue_slider">
         <property name="minimum">
          <number>-45</number>
         </property>
         <property name="maximum">
          <number>45</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_2">
         <property name="minimum">
          <number>-45</number>
         </property>
         <property name="maximum">
          <number>45</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>saturation  :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="saturation_slider">
         <property name="maximum">
          <number>60</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>17</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_3">
         <property name="maximum">
          <number>60</number>
         </property>
         <property name="value">
          <number>17</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>contrast      :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="contrast_slider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>32</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>14</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_4">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>32</number>
         </property>
         <property name="value">
          <number>14</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>411</x>
      <y>70</y>
      <width>269</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="play_btn">
       <property name="text">
        <string>play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_btn">
       <property name="text">
        <string>stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reset_btn">
       <property name="text">
        <string>reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>brightness_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>brightness_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>666</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hue_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>659</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hue_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturation_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>668</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>saturation_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>667</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>612</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contrast_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>669</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>contrast_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
